checks/checkforcontent.go

```diff
@@ -15,17 +15,14 @@
 package checks
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged